google/cloud/storage/benchmarks/benchmark_utils.cc

```diff
@@ -149,6 +149,15 @@ std::string MakeRandomBucketName(google::cloud::internal::DefaultPRNG& gen) {
   return storage::testing::MakeRandomBucketName(gen, RandomBucketPrefix());
 }
 
+std::string MakeRandomObjectName(google::cloud::internal::DefaultPRNG& gen) {
+  auto constexpr kObjectNameLength = 32;
+  return google::cloud::internal::Sample(gen, kObjectNameLength,
+                                         "abcdefghijklmnopqrstuvwxyz"
+                                         "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
+                                         "0123456789") +
+         ".txt";
```

Comment:
wouldn't this be a file name?

Reply:
I am not sure what you are asking.  But there are no files in GCS.  Just buckets and objects.

Reply:
I was just curious why there is a `".txt"` on the end.

Why not use: https://github.com/googleapis/google-cloud-cpp/blob/e2dd43461df4eb213e827653ca2c5b50657a8b0f/google/cloud/storage/testing/random_names.cc#L48-L58